llvm/unittests/TargetParser/TargetParserTest.cpp

```diff
@@ -1016,7 +1017,8 @@ TEST(TargetParserTest, ARMPrintSupportedExtensions) {
 
   outs().flush();
   testing::internal::CaptureStdout();
-  ARM::PrintSupportedExtensions();
+  std::map<StringRef, StringRef> EmptyMap;
```

Comment:
All tests for this should include at least 1 description, "this is a description" is fine. Just to prove we don't crash.

Reply:
Will address this, even by adding a dummy description (to skip whole target-machine creation +feature-query).

Reply:
Yeah that's fine. All it should check is that there's no path where a lookup (successful or not) causes an exception.